docker/build-local.sh

```diff
@@ -15,6 +17,7 @@ WORK_DIR="$(mktemp -d)"
 cp ../presto-server/target/presto-server-${PRESTO_VERSION}.tar.gz ${WORK_DIR}
 tar -C ${WORK_DIR} -xzf ${WORK_DIR}/presto-server-${PRESTO_VERSION}.tar.gz
 rm ${WORK_DIR}/presto-server-${PRESTO_VERSION}.tar.gz
+rdfind -makehardlinks true -makeresultsfile false ${WORK_DIR}
```

Comment:
It would be more portable to run this inside a docker, where you can assume everything about env & available packages. Oh, and it would "just work".

Of course you can't `COPY + RUN` in one docker layer, but the way to do this is a multi-stage build (https://docs.docker.com/develop/develop-images/multistage-build/#use-multi-stage-builds)

something like
```
FROM something
COPY server.tgz
RUN set -xeu; \
     apt-get install rdfind; \
     unpack tgz; \
     rdfind ...

FROM current_base
COPY --from=0(or some label) /some/path ...
```

Reply:
> Wouldn’t this cause the image to be larger?

If docker can retain hardlinks when copying from host to docker, then i assume it can too when copying from docker to docker.

> Could we leave this as a future enhancement?

yes
I'd suggest a TODO note to set the direction ("just works") we want to go to